app/dao/notifications_dao.py

```diff
@@ -136,6 +135,28 @@ def dao_get_7_day_agg_notification_statistics_for_service(service_id,
 
 
 @statsd(namespace="dao")
+def dao_get_template_usage(service_id, limit_days=None):
```

Comment:
Good call on this. Done:

  table = NotificationHistory

    if limit_days and limit_days < 7:  # can get this data from notifications table
        table = Notification